sql/monthly-revenue-query.sql

```diff
@@ -161,14 +161,39 @@ with conversions as (select
     CASE
         WHEN (fc.type ilike 'organization') THEN 1
         ELSE 0
-    END as "isOrg"
+    END as "isOrg",
+
+    /** isNotRefund: The transaction isn't either a refund or
+        refunded. */
+    CASE
+        WHEN (t."RefundTransactionId" IS NULL) THEN 1
+        ELSE 0
+    END as isNotRefund,
```

Comment:
Do we need `isNotRefund` at all?

Reply:
I kinda described the `isNotRefund` filter in the comment above, let me know if that doesn't make sense.